client/package.json

```diff
@@ -2,6 +2,7 @@
   "name": "templates",
   "version": "0.1.0",
   "private": true,
+  "homepage": ".",
```

Comment:
This is a trick to resolve relative path issue: https://create-react-app.dev/docs/deployment/#serving-the-same-build-from-different-paths